app/src/main/java/com/android/unio/ui/user/UserClaimAssociation.kt

```diff
@@ -66,39 +72,31 @@ fun UserClaimAssociationScreen(
         ) {
           Column(
               modifier = Modifier.padding(16.dp),
-              verticalArrangement = Arrangement.spacedBy(6.dp),
+              verticalArrangement = Arrangement.spacedBy(12.dp),
               horizontalAlignment = Alignment.CenterHorizontally,
           ) {
             Text(
-                context.getString(R.string.user_claim_association_you_can_either),
+                context.getString(R.string.user_claim_association_you_can_do_the_following),
                 style = AppTypography.headlineSmall)
 
             Text(
-                context.getString(R.string.user_claim_association_create_new_association),
+                context.getString(R.string.user_claim_association_claim_president_rights),
                 style = AppTypography.bodySmall)
 
-            Button(
-                onClick = {
-                  ToastUtils.showToast(
-                      context,
-                      context.getString(R.string.user_claim_association_not_implemented_yet))
-                },
-                modifier = Modifier.testTag(UserClaimAssociationTestTags.NEW_ASSOCIATION_BUTTON)) {
-                  Text(context.getString(R.string.user_claim_association_create_association))
-                }
+            val focusRequester = remember { FocusRequester() }
```

Comment:
It's a good idea to use the focusRequester ! I checked the doc about it and I think you are right to use it now ; )